src/snowflake/snowpark/functions.py

```diff
@@ -926,9 +926,34 @@ def covar_samp(
     return builtin("covar_samp", _emit_ast=_emit_ast)(col1, col2)
 
 
+def _check_map_parameter(*cols: typing.Any) -> None:
+    """Helper function to check parameter cols for create_map function."""
```

Comment:
we can move this function inside `_check_map_parameter` since no one else will be using it.

Reply:
moved to utils like other checks